### ISSUE TITLE
Proposed changes to theme_2dii_ggplot()

### ISSUE THREAD

**Issue by maurolepore** [2DegreesInvesting/r2dii.plot#110]
* Can we remove the suffix "ggplot" from the name? So we have `theme_2dii()` or similar.
* Should we pass `...` to `ggplot2::theme()`? As is this function is quite rigid. For example, the user may want to tweak the size of the font or some other detail.

**Comment by MonikaFu** [comment]
I agree with the first remark definitely. As for the second, I'm not familiar with how ... works so hard for me to say if it is a good idea and how would it work in practice. I think that if they want to tweak a specific detail, they can also do it after calling `theme_2dii()`

**Comment by maurolepore** [comment]
RE second bullet, here is an example -- we can discuss more in a call.

``` r
library(ggplot2)

my_theme <- function(size = 12, ...) {
  ggplot2::theme(text = element_text(size = size, ...))
}

p <- ggplot(mtcars) + geom_histogram(aes(disp), bins = 10)

# Use all defaults
p + my_theme()
```

![](https://i.imgur.com/WkV2eyM.png)

``` r
# Use explicit argument
p + my_theme(size = 40)
```

![](https://i.imgur.com/RUyL5K8.png)

``` r
# Use argument passed via ... to element_text()
p + my_theme(color = "blue")
```

![](https://i.imgur.com/xXvov9c.png)

<sup>Created on 2021-05-04 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by MonikaFu** [comment]
Closed by #168
